kotlin/android/app/src/main/res/layout/activity_session.xml

```diff
@@ -51,14 +51,38 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/llContainer" />
 
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/tabLayout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintTop_toBottomOf="@+id/tvResourcesList"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:tabInlineLabel="true">
+
+        <com.google.android.material.tabs.TabItem
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:icon="@drawable/baseline_star_24"
+            android:text="@string/resources_favorites" />
+
+        <com.google.android.material.tabs.TabItem
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:icon="@drawable/all_resources"
+            android:text="@string/resources_all" />
+
+    </com.google.android.material.tabs.TabLayout>
```

Comment:
This isn't a blocking concern but more of a question.

I don't really understand the choice of tabs here. To me, tabs are useful to separate unrelated entities, like Messages, Calls and Status in Whatsapp. 

Favourite and non-favourite resources are still resources. Why split them into different tabs? In Fastmail for example, I can "Pin" emails (which is basically like a favourite). Those are sticky at the top of all my emails and the rest is sorted by "Recevied" date.

Reply:
Absolutely no reason, I just had the tabs half-way implemented and I forgot what the conclusion was for UX.

@Patticatti If the pinned / sticky favorites sounds good, I'll change it to that before merging. Your call as UX person